### PR TITLE
Split out Figure 2A (and adjust Relate results)

### LINKED ISSUE #48: Figure 2A redundant?
Now that Figure 1 is much simpler, I wonder if Figure 2A (showing weights propagated up the tree) is needed. We only refer to it fleetingly in the text, and the caption only barely refers to it at all. Most of the caption is talking about 2B, then it talks about 2A and goes back to 2B. Further, readers might be confused about why we're omitting the weights in the right subtree and might think this is somehow significant.

I think the simplest thing is to remove 2A entirely and just make this an example of computing divergence. We can mention in the caption that weights are vector valued here.

### PR DESCRIPTION
**Pull request by petrelharp**
This closes #48. I may have messed up the flow; will definately need a read-through.

I've also added a new supplementary figure based on a different tree sequence from Leo: our previous assertion that Site/Branch hovered around the mutation rate was worrisome, since he thought the mutation rate should be 1.45e-8; this should clear that up and add some more nuance to that section.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Looks like we're missing the ``adding_weights`` figure here @petrelharp?

File: treestats_paper.tex [hunk 1]
```diff
@@ -1066,11 +1093,18 @@ \section*{Application to 1000 Genomes tree sequences}
...
-is relatively constant, hovering around typical estimates for the average genome-wide human mutation rate~\citep{narasimhan2017estimating}.
-On the one hand, this is expected ---
-Relate estimates branch lengths in part assuming that mutations fall at a constant rate through time
-(although \citeauthor{speidel2019method} also showed signal of changing mutation rates,
-pointing the way towards a more general method).
+is relatively constant, hovering around $2.5 \times 10^{-8}$.
+This is somewhat higher than
+typical estimates for the average genome-wide human mutation rate~\citep[e.g., $1.45\times 10^{-8}$,][]{narasimhan2017estimating},
+which suggests a mis-calibration in inferred tree times.
+These trees were inferred using Relate with the Ne fixed to 20,000;
```
Comment (jeromekelleher) [inline treestats_paper.tex:1100]: $N_e$ rather than Ne maybe? Also just below.

**Comment by petrelharp** [comment]
Whoops - actually put in the figure now. =)

**Comment by petrelharp** [comment]
> the new figures are super clear.

Will merge; thanks!

**Pull request merged**